### DISCUSSION TITLE
Only run selected honeypots?

### DISCUSSION THREAD

**Discussion by preservedfish** [telekom-security/tpotce#1416] [Q&A] [answered]
I'm running T-Pot in AWS and I'm hoping to cut down on resource usage so that I can switch to a cheaper instance. Would it be possible to only run one or two honeypots of my choosing?

**Comment by t3chn0m4g3** [comment, marked as answer]
You can adjust `tpot.yml` accordingly.

**Reply by encient** [reply]
Hi, may I know does it work the same for version 24.04? `tpot.yml` that I found in `tpotce/installer` doesn't seem to have honeypots listed. Or do we just comment out the section in `docker-compose.yml` just like disabling Ewsposter service?
